core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -252,25 +279,35 @@ var TopicEditorPage = function() {
   };
 
   this.dragSkillFromSubtopicToUncategorized = async function(
-      subtopicIndex, skillCardIndex) {
-    var subtopicCol = await subtopicColumns.get(subtopicIndex);
-    skillNamesElems = subtopicCol.all(
+      skillCardIndex) {
+    var skillNamesElems = element.all(
       by.css('.protractor-test-assigned-skill-card-text'));
     var toMove = await skillNamesElems.get(skillCardIndex);
     await dragAndDrop(toMove, uncategorizedSkillsContainer);
   };
 
-  this.expectSubtopicToHaveSkills = async function(subtopicIndex, skillNames) {
-    var subtopicCol = await subtopicColumns.get(subtopicIndex);
-    var skillNamesElems = subtopicCol.all(
+  this.navigateToTopicEditorTab = async function() {
+    var topicEditorTab = element(by.css('.protractor-test-edit-topic-tab'));
+    await waitFor.elementToBeClickable(topicEditorTab,
+      'Topic editor tab taking too long to be clickable');
+    await topicEditorTab.click();
+  };
+  this.navigateToSubtopicWithIndex = async function(subtopicIndex) {
+    var subtopic = await subtopics.get(subtopicIndex);
+    await subtopic.click();
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.expectSubtopicToHaveSkills = async function(skillNames) {
+    var skillNamesElems = element.all(
       by.css('.protractor-test-assigned-skill-card-text'));
     var skillNamesCount = await skillNamesElems.count();
     for (var i = 0; i < skillNamesCount; i++) {
       var skillCardTextElem = await skillNamesElems.get(i);
       var text = await skillCardTextElem.getText();
-      expect(skillNames[i]).toEqual(text);
+      await expect(skillNames[i]).toEqual(text);
     }
-    expect(skillNamesCount).toEqual(skillNames.length);
+    await expect(skillNamesCount).toEqual(skillNames.length);
```

Comment:
Why add these `await`s? I wouldn't have thought we needed them, so please let me know if I'm mistaken!

Reply:
No sorry(My IDE was suggesting them, forgot to remove them), Removed them.